client/src/partials/price_list/pricelist.html

```diff
@@ -1,27 +1,28 @@
 <div class="flex-header">
   <div class="bhima-title">
     <ol class="headercrumb">
-      <li class="static">{{ "TREE.ADMIN" | translate }}</li>
-      <li class="title">{{ "PRICE_LIST.TITLE" | translate }}</li>
+      <li class="static" translate> TREE.ADMIN </li>
+      <li class="active" translate> PRICE_LIST.TITLE </li>
     </ol>
-  </div>
-</div>
 
-<div class="flex-util">
-  <button class="btn btn-sm btn-default" id="create" ng-click="PriceListCtrl.create()" data-method="create">
-    <span class="fa fa-plus"></span> {{ "PRICE_LIST.NEW_PRICE_LIST" | translate }}
-  </button>
+   <div class="toolbar">
+     <div class="toolbar-item">
+      <button class="btn btn-sm btn-default" id="create" ng-click="PriceListCtrl.create()" data-method="create">
+        <span class="fa fa-plus"></span><span translate> {{ "PRICE_LIST.NEW_PRICE_LIST" | translate }} </span>
```

Comment:
If you could put a space between the `</span> <span translate>`, it would look nicer.

Also, because you have the `translate` directive, you don't need the filter.  You can just do this:
```html
<span class="fa fa-plus"></span><span translate>PRICE_LIST.NEW_PRICE_LIST</span>
```